openslides_backend/action/assignment/create_update_delete.py

```diff
@@ -8,10 +8,9 @@
 from ..generics import DeleteAction, UpdateAction
 from ..register import register_action_set
 
-create_schema = DefaultSchema(Assignment()).get_create_schema(
-    properties=[
-        "title",
-        "meeting_id",
+create_schema = DefaultSchema(Assignment()).get_default_schema(
```

Comment:
Here we can use `get_create_schema` which may use `get_default_schema` internally but readd title and description.